server/grpc_service.go

```diff
@@ -1621,7 +1621,12 @@ func (s *GrpcServer) ScatterRegion(ctx context.Context, request *pdpb.ScatterReg
 		return nil, err
 	}
 	if op != nil {
-		rc.GetOperatorController().AddOperator(op)
+		if !rc.GetOperatorController().AddOperator(op) {
+			return &pdpb.ScatterRegionResponse{
+				Header: s.wrapErrorToHeader(pdpb.ErrorType_UNKNOWN,
+					"operator cancelled"),
```

Comment:
How aboud `exceeded store limit causing operator canceled`?

Reply:
I don't think this message allows caller to determine why.

Reply:
@mittalrishabh You may need to resolve this comment before getting approved and merged.

Reply:
created a PR https://github.com/pingcap/kvproto/pull/1173

Reply:
I think you could just adjust the error message "operator cancelled" here.